src/views/UserDetail/UserDetail.js

```diff
@@ -309,6 +338,53 @@ class UserDetail extends React.Component {
     );
   }
 
+  showOpenTransactionsModal(json) {
+    this.setState({
+      showOpenTransactionModal: true,
+      openTransactions: json,
+    });
+  }
+
+  showDeleteUserModal(json) {
+    this.setState({
+      showDeleteUserModal: true,
+      openTransactions: json,
+    });
+  }
+
+  doClose = () => {
+    this.setState({
+      showDeleteUserModal: false,
+      showOpenTransactionModal: false,
+    });
+  }
+
+  selectModal(transactions) {
+    if (!transactions.hasOpenTransactions) {
+      this.showDeleteUserModal();
+    } else {
+      this.showOpenTransactionsModal(transactions);
+    }
+  }
+
+  doFetch() {
+    const { stripes } = this.props;
+    const okapiUrl = stripes.okapi.url;
+    const userId = this.props.match.params.id;
+
+    fetch(`${okapiUrl}/bl-users/by-id/${userId}/open-transactions`, {
```

Comment:
Is it possible to specify the GET open-transactions request as a mutator in \<UserRecordContainer\>? 

It may look like:
 ```
openTransactions: {
      type: 'okapi',
      path: 'bl-users/by-id/:{id}/open-transactions',
      accumulate: true
    },'
```
Then all requests are specified in one place. In addition, you wouldn't need to specify the `httpHeaders` in this file.